lib/matplotlib/widgets.py

```diff
@@ -2139,6 +2148,14 @@ class LassoSelector(_SelectorWidget):
     *onselect* : function
         Whenever the lasso is released, the `onselect` function is called and
         passed the vertices of the selected path.
+    *persist*: boolean
+        Whether to leave the lasso drawn until the next one is drawn.
+
+    *state_modifier_keys* are keyboard modifiers that affect the behavior
+        of the widget.
+
+        The defaults are:
+        dict(square='shift', clear='escape')
```

Comment:
of course, the name "square" is already in the API, but in isolation from other parts of the documentation, the name is not self-evident for what it does.